kong/db/schema/entities/consumers.lua

```diff
@@ -0,0 +1,18 @@
+local typedefs = require "kong.db.schema.typedefs"
+
+return {
+  name        = "consumers",
+  primary_key = { "id" },
```

Comment:
here we need to add ` endpoint_key = "username",` (see feat/admin-schema-aware-parser)